doc/modules/clustering.rst

```diff
@@ -54,6 +54,13 @@ Overview of clustering methods
      - General-purpose, even cluster size, flat geometry, not too many clusters
      - Distances between points
 
+   * - :ref:`K-Medoids <k_medoids>`
+     - number of clusters
+     - Large ``n_samples``, medium ``n_clusters``
```

Comment:
Large ``n_samples``? Doesn't is scale quadratically?

Reply:
done by Kornel